platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/PlatformStatus.java

```diff
@@ -24,11 +24,21 @@
  */
 public enum PlatformStatus implements UniqueId {
     /**
-     * The Platform is still starting up. This is the default state before ACTIVE
+     * The Platform is starting up.
      */
     STARTING_UP(1),
     /**
-     * All is normal: the Platform is running, connected, and syncing properly
+     * The platform is replaying events from the preconsensus event stream.
+     * (Not yet in use)
+     */
+    REPLAYING_EVENTS(7),
```

Comment:
Are these new statuses intentionally out of numerical order?

Reply:
Yes, I put them in logical order (to me at least)